cirq/sim/density_matrix_simulator.py

```diff
@@ -30,6 +30,14 @@
     from typing import Any, Hashable
 
 
+class _StateAndBuffers:
+
+    def __init__(self, n: int, state: np.ndarray):
```

Comment:
i'd call the inner state "matrix" to remind people it's the density matrix, but that's a nit.

Reply:
Done.